benchmarks/views/user.py

```diff
@@ -153,7 +201,77 @@ def post(self, request):
         return render(request, 'benchmarks/success.html')
 
 
-def resubmit(request):
+def validate_zip(file):
```

Comment:
@kvfairchild: can we import a method from `core` to do this and avoid redundany?